flexget/plugins/input/filmweb_watchlist.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils.cached_input import cached
+from flexget.entry import Entry
+
+try:
+    from filmweb.filmweb import Filmweb as FilmwebAPI
+    from filmweb.items import LoggedUser
+    from filmweb.exceptions import RequestFailed
+except ImportError:
+    pass
+else:
+    @event('plugin.register')
+    def register_plugin():
+        plugin.register(FilmwebWatchlist, 'filmweb_watchlist', api_ver=2)
+
+
+log = logging.getLogger('filmweb_watchlist')
+
+
+class FilmwebWatchlist(object):
+    """"Creates an entry for each movie in your Filmweb list."""
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'login': {'type': 'string', 'description': 'Can be username or email address'},
+            'password': {'type': 'string'},
+            'min_star': {
+                'type': 'integer',
+                'default': 0,
+                'description': 'Items will be processed with at least this level of "How much I want to see"'
+            }
+        },
+        'additionalProperties': False,
+        'required': ['login', 'password'],
+    }
+
+    @cached('filmweb_watchlist', persist='2 hours')
+    def on_task_input(self, task, config):
+        log.verbose('Retrieving filmweb watch list for user: %s', config['login'])
+
+        fw = FilmwebAPI()
+        log.verbose('Logging as %s', config['login'])
+
+        try:
+            fw.login(str(config['login']), str(config['password']))
+        except RequestFailed as error:
+            raise plugin.PluginError('Authentication request failed, reason %s' % str(error))
+
+        user = LoggedUser(fw)
+
+        try:
+            watch_list = user.get_want_to_see()
+        except RequestFailed as error:
+            raise plugin.PluginError('Fetching watch list failed, reason %s' % str(error))
+
+        log.verbose('Filmweb list contains %s items', len(watch_list))
+
+        entries = []
+        for item in watch_list:
+            if item['level'] < config['min_star']:
+                continue
+
+            item_info = item['film'].get_info()
+
+            # TODO: We are only supporting films.
+            # We can break from here, because film type appears first and it is sorted.
```

Comment:
Not a very good assumption. What if something changes and it's no longer sorted or sorted differently?

Reply:
You are right.  Fixed